panther_lights/src/animation/image_animation.py

```diff
@@ -41,11 +41,8 @@ def __init__(self, animation_description: dict, num_led: int, controller_freq: f
 
         # resize image to match duration
         original_img = imageio.imread(img_path)
```

Comment:
```suggestion
        original_img = Image.open(img_path)
```
This would save dependency on `imageio`